docs/cloud-soar/automation.md

```diff
@@ -81,8 +81,6 @@ Watch this micro lesson to learn how to create custom playbooks.
 
 To configure a new Playbook, click the cog icon (<img src={useBaseUrl('img/cloud-soar/cog.png')} alt="cog menu" width="20"/>) > **Automation**.
 
-![Playbook](/img/cloud-soar/image71.png)
```

Comment:
@jpipkin1 this img was deleted without replacement code - was this intentional?

Reply:
Yep.